cardano-cli/src/Cardano/CLI/Run/Hash.hs

```diff
@@ -118,6 +121,29 @@ runHashAnchorDataCmd Cmd.HashAnchorDataCmdArgs{toHash, mExpectedHash, mOutFile}
         mkHandler :: (Monad m, Exception e) => (e -> HttpRequestError) -> Handler m HashCmdError
         mkHandler x = Handler $ return . HashGetFileFromHttpError . x
 
+    convertToHttp :: URI -> ExceptT HashCmdError IO URI
```

Comment:
I would move this to the top level. In subsequent PRs you will be calling this code in any case.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/895/commits/77430a66d9bb2349c310cb0033482c634ae0195e

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/pull/895/commits/b07b0e5f4c68c201718252506a3b89c33a0fdb79